src/test/kotlin/no/brreg/informasjonsforvaltning/abackendservice/utils/TestData.kt

```diff
@@ -17,21 +20,33 @@ const val MONGO_PORT = 27017
 const val MONGO_USER = "testuser"
 const val MONGO_PASSWORD = "testpassword"
 
-val GENERATED_ID_0 = ObjectId("5d846c475f599c04093216c3")
+const val EXISTING_SERVICE="a-duplicate-service"
+
+val MONGO_ENV_VALUES: Map<String, String> = ImmutableMap.of(
+        "MONGO_INITDB_ROOT_USERNAME", MONGO_USER,
+        "MONGO_INITDB_ROOT_PASSWORD", MONGO_PASSWORD
+)
 
-const val DATABASE_NAME = "a-backend-service"
-const val MONGO_COLLECTION = "service-endpoints"
-private const val MONGO_AUTH = "?authSource=admin&authMechanism=SCRAM-SHA-1"
+val API_ENV_VALUES : Map<String,String> = ImmutableMap.of(
+        "MONGO_USERNAME", MONGO_USER,
+        "MONGO_PASSWORD", MONGO_PASSWORD,
+        "MONGO_HOST", "$MONGO_SERVICE_NAME:$MONGO_PORT",
+        "SPRING_PROFILES_ACTIVE" , "test"
+)
 
-const val ADMIN_TOKEN_TMP ="dfghjklasfijkf5fgahfskjl"
-const val NON_ADMIN_TOKEN_TMP ="dfghjklasfijkf5fgahfskjl"
 
+const val LOCAL_SERVER_PORT = 5000
+
+val GENERATED_ID_0 = ObjectId("5d846c475f599c04093216c3")
 const val SERVICE_ENDPOINT = "/serviceendpoints"
+const val VERSION_API_ENDPOINT = "/version"
+
+fun getApiAddress( endpoint: String ): String{
+   return "http://${TEST_API.getContainerIpAddress()}:${TEST_API.getMappedPort(API_PORT)}$endpoint"
+}
 
-fun mongoUri(host: String, port: Int): String =
-    "mongodb://$MONGO_USER:$MONGO_PASSWORD@$host:$port/$DATABASE_NAME$MONGO_AUTH"
 
-fun createServiceEndpointDB(testName: String, testUrl: String) =
+fun createServiceEndpointDB(testName: String,testUrl: String) =
```

Comment:
vi skal ikke gå for editorconfig elns for å opprettholde coding style?

Reply:
Jo!